homeassistant/components/gpstracker/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""The GPS Tracker integration."""
+from __future__ import annotations
+
+import gps_tracker
+
+from homeassistant.config_entries import (
+    ConfigEntry,
+    ConfigEntryAuthFailed,
+    ConfigEntryNotReady,
+)
+from homeassistant.const import CONF_ENTITIES, CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CLIENT, DOMAIN
+from .helpers import get_gpstracker_client
+
+PLATFORMS: list[Platform] = [Platform.DEVICE_TRACKER]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up GPS Tracker from a config entry."""
+    hass.data.setdefault(DOMAIN, {}).setdefault(entry.entry_id, {})
+
+    config = gps_tracker.Config(  # type: ignore[call-arg]
+        password=entry.data.get(CONF_PASSWORD),
+        username=entry.data.get(CONF_USERNAME),
```

Comment:
If it is compulsory, don't use .get() (same for password)
```suggestion
        username=entry.data[CONF_USERNAME],
```

Reply:
Fixed in 6cd945738e66a249e8691c243d817e869028b682